sdk/src/main/java/com/horizen/box/data/CoreBoxesDataIdsEnum.java

```diff
@@ -0,0 +1,18 @@
+package com.horizen.box.data;
+
+public enum CoreBoxesDataIdsEnum {
```

Comment:
if we have one to one relation between data and boxes, why not use the same enum everywhere?